presentation/src/main/java/org/sagebionetworks/research/presentation/model/form/InputFieldView.java

```diff
@@ -61,9 +61,10 @@
     String getFormUIHint();
 
     /**
-     * @return identifier that is unique among form items within the step
+     * @return A short string that uniquely identifies the input field within the step. The identifier is
+     *         reproduced in the results of a step result in the step history of a task result.
      */
-    @Nullable
+    @NonNull
```

Comment:
iOS has this as non-null, and it takes on the identifier of the form step if none is provided.  @syoung-smallwisdom can you verify?

Reply:
Yup.